packages/calcite-components/src/components/functional/Validation.tsx

```diff
@@ -21,8 +22,12 @@ export const Validation = ({
   id,
   icon,
   message,
+  ref,
 }: ValidationProps): TemplateResult => (
-  <div class={CSS.validationContainer}>
+  <div
+    class={CSS.validationContainer}
+    ref={ref ? ref : (referenceElement): HTMLDivElement => referenceElement}
```

Comment:
Can you extract the ref-handling logic to a top-level function instead of using an anonymous function? The latter will create a new function on each render.